perceval/backends/weblate/weblate.py

```diff
@@ -291,14 +291,13 @@ def changes(self, from_date=None):
         """Fetch changes of a project."""
 
         payload = {}
-
         if from_date:
             payload[self.PAFTER] = from_date.isoformat()
 
         if self.project:
-            path = urijoin(self.base_url, 'projects', self.project, 'changes')
+            path = urijoin(self.base_url, 'projects', self.project, 'changes', '')
```

Comment:
```suggestion
            path = urijoin(self.base_url, 'projects', self.project, 'changes') + '/'
```

Reply:
I think this is more clear. Maybe you can add a comment before the `if` block explaining why this is needed.